.vscode/settings.json

```diff
@@ -8,6 +8,7 @@
     },
     // Turn off tsc task auto detection since we have the necessary tasks as npm scripts
     "typescript.tsc.autoDetect": "off",
+    "prettier.tabWidth": 2,
```

Comment:
I have another default and on every save I would reformat the entire document. s1."what the best practice is here but specifying formatter settings per project seems like a good idea?

Reply:
Sounds good

Reply:
I will revert this change and we can make a follow up pr